assets/js/googlesitekit/data/create-error-store.js

```diff
@@ -92,12 +101,31 @@ export function createErrorStore() {
 
 			case CLEAR_ERROR: {
 				const { baseName, args } = payload;
-				const key = generateErrorKey( baseName, args );
+				const newState = { ...state };
+				if ( baseName ) {
+					const key = generateErrorKey( baseName, args );
+					newState.errors = { ...( state.errors || {} ) };
+					delete newState.errors[ key ];
+				} else {
+					// @TODO: remove it once all instances of the legacy behavior have been removed.
+					delete newState.error;
```

Comment:
I don't think we should `delete` full state properties.
```suggestion
					newState.error = undefined;
```